cloud/azure/modules/app/locals.tf

```diff
@@ -79,6 +79,7 @@ locals {
 
     CIVIFORM_API_SECRET_SALT = data.azurerm_key_vault_secret.api_secret_salt_key.value
 
+    CIVIFORM_ADMIN_REPORTING_UI_ENABLED          = var.feature_flag_reporting_enabled
```

Comment:
Where is this used?

Reply:
It's setting the env var for the server

Reply:
I see, I missed it is in the `app_settings` object